taxi/models.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+from django.conf import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=63, unique=True)
+    country = models.CharField(max_length=63)
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=63, unique=True)
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=63)
+    manufacture = models.ForeignKey(
+        Manufacturer,
+        on_delete=models.CASCADE,
+        related_name="cars"
```

Comment:
The `related_name` for the `manufacture` ForeignKey is correctly set to 'cars', which is appropriate for the `Manufacturer` model.